hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/SnapshotScannerHDFSAclHelper.java

```diff
@@ -71,23 +72,23 @@
 public class SnapshotScannerHDFSAclHelper implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(SnapshotScannerHDFSAclHelper.class);
 
-  public static final String USER_SCAN_SNAPSHOT_ENABLE = "hbase.user.scan.snapshot.enable";
-  public static final String USER_SCAN_SNAPSHOT_THREAD_NUMBER =
-      "hbase.user.scan.snapshot.thread.number";
+  public static final String ACL_SYNC_TO_HDFS_ENABLE = "hbase.acl.sync.to.hdfs.enable";
```

Comment:
better to name it as 'hbase.table.acl.sync.to.hdfs.enable'  ?

Reply:
The steps to config this feature is as followings:
1. Check HDFS configuration
2. Add master coprocessor:
    hbase.coprocessor.master.classes =       
    “org.apache.hadoop.hbase.security.access.AccessController
    ,org.apache.hadoop.hbase.security.access.SnapshotScannerHDFSAclController”
3. Enable this feature:
    hbase.acl.sync.to.hdfs.enable=true
4. Modify table scheme to enable this feature:
    alter 't1', CONFIGURATION => {'hbase.acl.sync.to.hdfs.enable' => 'true'}

So this config has two usages in both step 3 and 4.
If step3 is set, the acl will aync to HDFS if grant namespace or global. 
Maybe 'hbase.table.acl.sync.to.hdfs.enable' is confused because global and namespace acls are also synced?

Reply:
OK